src/Components/Web/src/WebRenderer.cs

```diff
@@ -51,15 +76,57 @@ protected internal int AddRootComponent([DynamicallyAccessedMembers(Component)]
         /// <param name="domElementSelector">A CSS selector that uniquely identifies a DOM element.</param>
         protected abstract void AttachRootComponentToBrowser(int componentId, string domElementSelector);
 
+        /// <inheritdoc />
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _interopMethodsReference.Dispose();
+            }
+
+            base.Dispose(disposing);
+        }
```

Comment:
For a change, we actually have an `unmanaged` resource (the DotNetObjectReference). If you really wanted you could use the Dispose pattern.